### PR TITLE
Pixabay: Get 1080p videos by default instead of 4k

### PR DESCRIPTION
**Pull request by mgbaybay**
## Changes
BE:
- Get 1080p videos by default instead of 4k

Relates to: https://github.com/xibosignageltd/xibo-private/issues/840

### CODE REVIEW

File: lib/Connector/PixabayConnector.php [hunk 1]
```diff
@@ -221,10 +226,6 @@ public function onLibraryProvider(LibraryProviderEvent $event)
                             $result->picture_id,
                             'https://i.vimeocdn.com/video/pictureId_960x540.png'
                         );
-                    } else {
```
Comment (dasgarner) [inline lib/Connector/PixabayConnector.php:224]: I am not clear why we moved from using the medium thumbnail with tiny thumbnail fallback, to using the thumbnail for each size of video. Aren't the "large" ones going to be too big for our usage of a thumbnail?

Comment (mgbaybay) [reply lib/Connector/PixabayConnector.php]: Thanks for this! I was thinking that there must be some sort of consistency (ie if we use large video, then we should use large thumbnail as well) haha 😄 but you're right, some thumbnails might be too large for our use case. 

Reverted the changes :)

**Review by dasgarner — APPROVED** [review]

**Review by ifarzana — APPROVED** [review]
Approving to build container

**Review by ifarzana — APPROVED** [review]
Approving to build container

**Comment by ifarzana** [comment]
TEST: /Library END

**Comment by ifarzana** [comment]
The error was expected due to a bug. We will run another test to ensure the PR gets the green light.

**Comment by ifarzana** [comment]
TEST: login.cy.js END

**Pull request merged**